tests/test_meteo.py

```diff
@@ -75,17 +76,23 @@ def test_meteo_passthrough():
     assert new_meteo.Dataset.equals(original_meteo.Dataset)
 
 
-@pytest.mark.xfail
 def test_meteo_url():
-    today = pd.to_datetime(datetime.datetime.today())
+    geometry = {
+        "lon_min": -25.0,  # lat/lon window
+        "lon_max": -9.0,
+        "lat_min": 56.0,
+        "lat_max": 74.0,
+    }
+    cdate = pd.to_datetime("today") - pd.DateOffset(days=1)  # step back one day for availability.
+    r = [0, 6, 12, 18]
+    h = np.argmin([n for n in [cdate.hour - x for x in r] if n > 0])
     url = "https://nomads.ncep.noaa.gov/dods/gfs_0p25_1hr/gfs{}/gfs_0p25_1hr_{:0>2d}z".format(
-        today.strftime("%Y%m%d"), r[h]
+        cdate.strftime("%Y%m%d"), r[h]
     )
-    meteo = pmeteo.meteo(meteo_source=url)
+    meteo = pmeteo.meteo(meteo_source=url, **geometry)
     assert isinstance(meteo.Dataset, xr.Dataset)
 
 
-@pytest.mark.xfail
-def test_meteo_default_arguments():
+def test_meteo_empty():
     meteo = pmeteo.meteo(meteo_source=None)
-    assert isinstance(meteo.Dataset, xr.Dataset)
+    assert meteo.Dataset == None
```

Comment:
I think I would add one more test here:
``` python
def test_meteo_defaults():
    meteo = pmeteo.meteo()
    assert meteo.Dataset == None
```

Reply:
How is it different from 

```
def test_meteo_empty():
    meteo = pmeteo.meteo(meteo_source=None)
    assert meteo.Dataset == None

```
?

Reply:
``` diff
+ pmeteo.meteo(meteo_source=None)
- pmeteo.meteo()
```
Τhe first time you explicitly pass `None` ως argument, while the second time you don't pass any arguments, therefore you get the default values, whatever these are.

In this case, it happens that the default argument is `None` so indeed the tests are equivalent. But this might change in the future. E.g. we might make `meteo_source` a mandatory argument or we might decide that a different default value is better suited to our API. In these cases, the test I propose will fail, while the one we currently have will still pass

Reply:
ok